en/5x/api.md

```diff
@@ -11,6 +11,8 @@ redirect_from: "/5x/api.html"
 
   {% include note.html content="This is early beta documentation that may be incomplete and is still under development." %}
 
+  {% include note.html content="Express 5.0 will require Node.js 18 or higher." %}
```

Comment:
"Express 5.o requires..."

Since 5.x is available now (in beta) it should be present tense.

Reply:
Thanks @crandmck !

Change added [here](https://github.com/expressjs/expressjs.com/pull/1495/commits/9631a68116f17db2c6a4271720dbf31c2eb8c3bf)